### PR TITLE
chore: extend Version Upgrade

### PR DESCRIPTION
**Pull request by Phil91**
## Description

add description for the company credential detail database changes

## Why

To have the latest changes visible in the Version Upgrade

## Issue

N/A - Jira Ticket: CPLP-2853

## Corresponding Backend PR

[#84](https://github.com/eclipse-tractusx/portal-backend/pull/84)

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md [hunk 1]
```diff
@@ -8,6 +8,25 @@ Each section includes the respective change details, impact on existing data and
...
+* NEW: portal.use_case_participation_status
+* NEW: portal.company_ssi_details
+* NEW: portal.verified_credential_type_assigned_use_cases
+
+New use_case_descriptions table released to be able to create translatable use case descriptions.
+New verified_credential_types, verified_credential_type_kinds & verified_credential_type_assigned_kinds tables to store the type of the credentials as well as the kind of the type and have a mapping between the type and kind available.
+New use_case_participation_status table to have static data of the status.
+New company_ssi_details table to be able to safe the credential details for a specific company.
+New verified_credential_type_assigned_use_cases to map the verified_credential_type to a specific use case.
+
```
Comment (jjeroch) [inline developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md:29]: @Phil91 I suggest we add the following picture
SSI Release 1 6 0 Introduction

can you please validate it?

File: developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md [hunk 2]
```diff
@@ -8,6 +8,25 @@ Each section includes the respective change details, impact on existing data and
...
+<br>
+
+#### Company Credential Details - NEW - x.x.x
+
+* NEW: portal.use_case_descriptions
+* NEW: portal.verified_credential_types
+* NEW: portal.verified_credential_type_kinds
+* NEW: portal.verified_credential_type_assigned_kinds
+* NEW: portal.use_case_participation_status
+* NEW: portal.company_ssi_details
```
Comment (jjeroch) [inline developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md:21]: @Phil91 I think we missed company_ssi_details_status - could you please check once

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md (hunk 1 above)
Comment (jjeroch) [reply developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md:29]: and likely the following as well
image

Comment (Phil91) [reply developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md:33]: on the first picture the table 'verified_credential_type_assigned_external_types' is missing. thats the linkage between 'verified_credential_types' and 'verified_credential_external_types'.

Where should i include the pictures in the folder structure?

File: developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md (hunk 2 above)
Comment (Phil91) [reply developer/Technical Documentation/Version Upgrade/portal-upgrade-details.md]: added the missing items

**Review by jjeroch — APPROVED** [review]

**Pull request merged**